okhttp-coroutines/build.gradle.kts

```diff
@@ -20,8 +20,9 @@ dependencies {
   api(libs.squareup.okio)
   api(libs.kotlin.stdlib)
 
-  testImplementation(libs.kotlin.test.common)
   testImplementation(libs.kotlin.test.annotations)
+  testImplementation(libs.kotlin.test.common)
+  testImplementation(libs.kotlin.test.junit)
```

Comment:
The need for all three of these is very strange. You should really only need "common" as the Gradle module metadata will resolve both the annotations and the junit implementation automatically in almost all cases.

There are a few exceptions, maybe in the MWS modules, where you have to be explicit. But 99% of modules should get the automatic behavior.

Reply:
Yeah, I’m frustrated with this one. For reasons unclear to me omitting `libs.kotlin.test.junit` causes compile to fail.

```
> Task :okhttp-brotli:compileTestKotlin FAILED
e: file:///Volumes/Development/okhttp/okhttp-brotli/src/test/java/okhttp3/brotli/BrotliInterceptorTest.kt:24:20 Unresolved reference: assertFailsWith
```